bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -120,10 +122,20 @@ export class BoxAnnotationView extends AnnotationView {
     ctx.restore()
   }
 
-  bbox_contains(sx: number, sy: number): boolean {
+  interactive_bbox() {
```

Comment:
`tslint` will complain. Every function and method must have explicit return type.

Reply:
right just an oversight

Reply:
fixed in https://github.com/bokeh/bokeh/pull/7839/commits/e7d2ef600b0a95c4bd67af7d3473a1aba7830c2a